Sources/CryptomatorCloudAccess/Dropbox/DropboxCloudProvider.swift

```diff
@@ -191,8 +191,8 @@ public class DropboxCloudProvider: CloudProvider {
 					return
 				}
 				if let networkError = networkError {
-					CloudAccessDDLogDebug("DropboxCloudProvider: fetchItemMetadata(at: \(cloudPath.path)) failed with networkError: \(networkError)")
-					reject(self.convertRequestErrorToDropboxError(networkError))
+					CloudAccessDDLogDebug("DropboxCloudProvider: fetchItemMetadata failed with networkError: \(networkError)")
```

Comment:
Revert the deleted `(at: \(cloudPath.path))` in debug logging.

Reply:
Will be reverted in the next commit.